core/src/main/java/com/scalar/db/sql/Example.java

```diff
@@ -0,0 +1,488 @@
+    resultSet.close();
+  }
+
+  public static void twoPhaseCommitTransactionModeExample(SqlSessionFactory sqlSessionFactory) {
+    System.out.println("Start two-phase commit transaction mode example");
+
+    StorageSqlSession storageSqlSession = sqlSessionFactory.getStorageSqlSession();
+
+    ResultSet resultSet;
+
+    // Create namespace
+    CreateNamespaceStatement createNamespaceStatement =
+        SqlBuilder.createNamespace(NAMESPACE_NAME).ifNotExists().build();
+    resultSet = storageSqlSession.execute(createNamespaceStatement);
+    resultSet.close();
+
+    createNamespaceStatement =
+        SqlBuilder.createNamespace(Coordinator.NAMESPACE).ifNotExists().build();
+    resultSet = storageSqlSession.execute(createNamespaceStatement);
+    resultSet.close();
+
+    // Create table
+    CreateTableStatement createTableStatement =
+        SqlBuilder.createTable(NAMESPACE_NAME, TABLE_NAME)
+            .ifNotExists()
+            .withPartitionKey(COLUMN_NAME_1, DataType.INT)
+            .withClusteringKey(COLUMN_NAME_2, DataType.TEXT)
+            .withColumn(COLUMN_NAME_3, DataType.BIGINT)
+            .withColumn(COLUMN_NAME_4, DataType.FLOAT)
+            .withColumn(Attribute.ID, DataType.TEXT)
+            .withColumn(Attribute.STATE, DataType.INT)
+            .withColumn(Attribute.VERSION, DataType.INT)
+            .withColumn(Attribute.PREPARED_AT, DataType.BIGINT)
+            .withColumn(Attribute.COMMITTED_AT, DataType.BIGINT)
+            .withColumn(Attribute.BEFORE_PREFIX + COLUMN_NAME_3, DataType.BIGINT)
+            .withColumn(Attribute.BEFORE_PREFIX + COLUMN_NAME_4, DataType.FLOAT)
+            .withColumn(Attribute.BEFORE_ID, DataType.TEXT)
+            .withColumn(Attribute.BEFORE_STATE, DataType.INT)
+            .withColumn(Attribute.BEFORE_VERSION, DataType.INT)
+            .withColumn(Attribute.BEFORE_PREPARED_AT, DataType.BIGINT)
+            .withColumn(Attribute.BEFORE_COMMITTED_AT, DataType.BIGINT)
+            .withClusteringOrder(COLUMN_NAME_2, Order.ASC)
+            .build();
+    resultSet = storageSqlSession.execute(createTableStatement);
+    resultSet.close();
+
+    createTableStatement =
+        SqlBuilder.createTable(Coordinator.NAMESPACE, Coordinator.TABLE)
+            .ifNotExists()
+            .withPartitionKey(Attribute.ID, DataType.TEXT)
+            .withColumn(Attribute.STATE, DataType.INT)
+            .withColumn(Attribute.CREATED_AT, DataType.BIGINT)
+            .build();
+    resultSet = storageSqlSession.execute(createTableStatement);
+    resultSet.close();
+
+    // Begin
+    TwoPhaseCommitTransactionSqlSession twoPhaseCommitTransactionSqlSession1 =
+        sqlSessionFactory.beginTwoPhaseCommitTransaction();
+
+    TwoPhaseCommitTransactionSqlSession twoPhaseCommitTransactionSqlSession2 =
+        sqlSessionFactory.joinTwoPhaseCommitTransaction(
+            twoPhaseCommitTransactionSqlSession1.getTransactionId());
```

Comment:
We use `TwoPhaseCommitTransactionSqlSession` for Two-phase Commit Transaction mode.